src/accounts/views.py

```diff
@@ -1,31 +1,92 @@
 # Create your views here.
-from django.contrib.auth import login
+import os
+from django.contrib.auth import login, logout
 from django.shortcuts import render, redirect
 from django.contrib.auth.decorators import login_required
+from django.contrib import messages
+from django.contrib.auth.models import User
+from django.contrib.messages import get_messages
+from django.db.models.signals import post_migrate
+from django.dispatch import receiver
+from allauth.socialaccount.models import SocialApp
+from dotenv import load_dotenv, set_key
 from .forms import MyAuthenticationForm, MyUserCreationForm
 
+# In your app's views.py or signals.py
+@receiver(post_migrate)
+def my_signal_receiver(sender, **kwargs): # pylint: disable=unused-argument
+    # Safe to access the database here
+
+    # Create Site
+    from django.contrib.sites.models import Site
+
+    site, _ = Site.objects.get_or_create(
+        domain=os.getenv('domain','http://127.0.0.1:8000'),
+        defaults={'name': os.getenv('domain_name', 'http://127.0.0.1:8000')}
+    )
+
+    site.save()
+
+    # Path to your .env file
+    load_dotenv()
+    env_file = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), '.env') # pylint: disable=locally-disabled, multiple-statements, fixme, line-too-long
```

Comment:
1/ I think you only need the `line-too-long` disabled. And rather than using a disable, you could just split this into two lines at the `'.env'`. 

```python
    env_file = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
                            '.env')
```

Reply:
Updated